task.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+trait ChangeAttributes {

```

Comment:
Each class, interface, trait should be in the separate php file. The name of this file should be equal to its name. And you have to use PSR-4 autoloader system and use statements instead of writing all your code in one file.